repositories.md

```diff
@@ -198,3 +198,9 @@ An OEM MUST define a public API to the image repository for  Primaries to use in
 
 Depending on the OEM's requirements, this API MAY require authentication before primaries are allowed to download updates. The OEM is free to use any authentication method. Such a choice affects only how certain the OEM can be that it is communicating with authentic primaries, and does not affect how resilient ECUs are to a compromise of the image repository.
 
+## Specifying wireline formats
+In setting up the Uptane program, an implementer will need to specify how information, such as cryptographic hashes, digital signatures, and public keys, should be encoded. As the guiding principle of the Uptane framework is to give each implementer as much design flexibility as possible, the Uptane Standard does not specify particular data binding formats. Instead, OEMs and suppliers can continue to use the protocols and formats of existing update systems, or they can select formats that best  ensure interoperability with other essential technologies. 
+
+To facilitate the use of these specifications, an Uptane implementer can choose to write a POUF, an added layer to the Standard in which an implementer can specify choices of Protocols, Operations, Usage and Formats. A POUF provides an easy way for an implementer to specify the elements that can ensure interoperability, and can be customized as needed for different makes and models of autos and/or for the special needs of fleet owners, and for police or military vehicles.
```

Comment:
```suggestion
To facilitate coordination between implementations, an Uptane adopter can choose to write a POUF, an added layer to the Standard in which an implementer can specify choices of Protocols, Operations, Usage and Formats. A POUF provides an easy way for an implementer to specify the elements that can ensure interoperability, and can be customized as needed for different makes and models of autos and/or for the special needs of fleet owners, and for police or military vehicles.
```

Reply:
Thanks, Marina. I accepted the suggestion.